src/lib/components/data-graphic/guides/Axis.svelte

```diff
@@ -173,12 +174,18 @@ This component will draw an axis on the specified side.
     // use graphicWidth or graphicHeight
     // do we ensure different spacing in one case vs. another?
     tickCount = ~~(axisLength / 20);
-    tickCount = Math.max(2, ~~(tickCount / 100));
+    const maxTickCountThreshold = 2 + ~~($plotConfig.width / 250);
```

Comment:
Could we generalize this code and also add it to `Grid.svelte` as well? Seems like this could be a function that takes in the arguments used in this. Otherwise the grid gets out of sync with the axis ticks

<img width="500" alt="Screen Shot 2022-07-25 at 7 03 25 AM" src="https://user-images.githubusercontent.com/95735/180796090-2a1a013a-e7a0-4807-aa2d-ba3770461468.png">

Reply:
Have put a generic method in `data-graphic/utils`